src/anyio/_backends/_asyncio.py

```diff
@@ -974,7 +975,10 @@ def run(self) -> None:
                             self._report_result, future, result, exception
                         )
 
+                    del result, exception
+
                 self.queue.task_done()
+                del item, context, func, args, future, cancel_scope
```

Comment:
item can be deleted up on line 961 after it's unpacked

Reply:
That just adds another code line with no benefit?